src/lib.rs

```diff
@@ -129,22 +130,27 @@ impl NwType {
     }
 }
 
+/// Represents a named value in the Neverwinter Nights module file format.
 #[derive(Debug, Serialize, Deserialize)]
 pub struct NwValue<T> {
     #[serde(rename = "type")]
-    pub _type: String,
+    pub _type: String, // Represents the type of the value
+
     #[serde(rename = "value")]
-    pub value: T,
+    pub value: T, // The actual value of type T
```

Comment:
What does this look like in doc.rs?

Reply:
It looks like someone doesn't know what they are doing. Fixing it.

Reply:
https://github.com/milliorn/rs_nwn_nasher_types/pull/9/commits/3fd4195b0b345d8a0a031b908ca6785847b26c4b